src/main/java/blackjack/view/CardShapeStrings.java

```diff
@@ -0,0 +1,36 @@
+package blackjack.view;
+
+import blackjack.domain.card.CardShape;
+
+public enum CardShapeStrings {
+    SPADE("스페이드"),
+    HEART("하트"),
+    DIAMOND("다이아몬드"),
+    CLOVER("클로버");
+
```

Comment:
다른 Enum값이 추가될 경우를 생각해서 다음처럼 쓰곤 합니다 :)
```suggestion
    DIAMOND("다이아몬드"),
    CLOVER("클로버"),
    ;
```